api/src/functions/processValidationJson/processValidationJson.ts

```diff
@@ -49,16 +80,23 @@ export const processRecord = async (
   const key = record.s3.object.key
 
   // Download the JSON errors file from S3
-  const getObjectResponse = await s3Client.send(
-    new GetObjectCommand({ Bucket: bucket, Key: key })
-  )
+  let getObjectResponse
+  try {
+    getObjectResponse = await s3Client.send(
+      new GetObjectCommand({ Bucket: bucket, Key: key })
+    )
+  } catch (err) {
+    logger.error(`Error getting object from S3: ${err} - key: ${key}`)
+    throw new Error('Error getting object from S3')
+  }
 
   // Add the contents of the JSON to the database
   if (getObjectResponse.Body) {
     /* example file path/key
          /uploads/organization_id/agency_id/reporting_period_id/upload_id/{filename}
       */
-    const result = JSON.parse(getObjectResponse.Body.toString()) || []
+    const strBody = await getObjectResponse.Body.transformToString()
+    const result = JSON.parse(strBody) || []
 
     // when the results array is empty then we know the file has passed validations
     const passed = result.length === 0
```

Comment:
Small clarification -- does this mean essentially if we can't parse the strBody to JSON above that we'll say it's passed? Since we're defaulting to an empty list